### ISSUE TITLE
Rule Change: Implement suggestions for no-empty-function

### ISSUE THREAD

**Issue by jaymarvelz** [eslint/eslint#20055]
### What rule do you want to change?

no-empty-function

### What change do you want to make?

Implement suggestions

### How do you think the change should be implemented?

Other

### Example code

```js
/*eslint no-empty-function: "error"*/

function foo() {}
```

### What does the rule currently do for this code?

Doesn't provide fixable suggestions to add comments

### What will the rule do after it's changed?

Provide a suggestion to add `/* empty */` comment inside function bodies

### Participation

- [x] I am willing to submit a pull request to implement this change.

**Comment by mdjermanovic** [comment]
Makes sense to me 👍 

@eslint/eslint-team thoughts?

**Comment by nzakas** [comment]
Sounds good. Marking as accepted.
